aptos-move/framework/aptos-framework/sources/managed_coin.spec.move

```diff
@@ -68,10 +67,20 @@ spec aptos_framework::managed_coin {
         amount: u64,
     ) {
         let account_addr = signer::address_of(account);
+
+        // Capabilities<CoinType> should not exist in the signer address.
         aborts_if !exists<Capabilities<CoinType>>(account_addr);
+
         let coin_store = global<coin::CoinStore<CoinType>>(dst_addr);
+        // Capabilities<CoinType> should not exist in the destination address and `dst_addr` should not be frozen.
         aborts_if !exists<coin::CoinStore<CoinType>>(dst_addr);
         aborts_if coin_store.frozen;
+
+        // Ensure that 'amount' is withdrawn from 'account'.
```

Comment:
Could you double check whether this property should hold for the mint function? @aalok-t

Reply:
By the way, did this pass the Prover test locally?

Reply:
Could you try `ensures false;` and see if Prover proves it? If so, it is an inconsistency problem.

Reply:
If it is not an inconsistency issue, it may be related to the spec of  `mint` in the `coin` module, which is opaque. @junkil-park